packages/paste-website/src/component-examples/VisualPickerExamples.ts

```diff
@@ -0,0 +1,828 @@
+                    <Avatar name="video recipient2" icon={UserIcon} />
+                    <Avatar name="video recipient3" icon={UserIcon} />
+                  </AvatarGroup>
+                  <Text fontWeight="fontWeightBold">+83</Text>
+                </Box>
+              </Box>
+            </VisualPickerCheckbox>
+            <VisualPickerCheckbox id={useUID()} labelText="email" checked={checked4} onChange={handleChange4}>
+              <Box display="flex" justifyContent="space-between" alignItems="center">
+                <MediaObject verticalAlign="center">
+                  <MediaFigure spacing="space50">
+                    <Avatar variant="entity" icon={EmailIcon} size="sizeIcon90" name="code" color="decorative20" />
+                  </MediaFigure>
+                  <MediaBody>
+                    <Text as="div" fontWeight="fontWeightSemibold">
+                      Email
+                    </Text>
+                    <Text as="div" color="colorTextWeak">
+                      6,081 recipients
+                    </Text>
+                  </MediaBody>
+                </MediaObject>
+                <Box display="flex" columnGap="space30">
+                  <AvatarGroup size="sizeIcon20" variant="user">
+                    <Avatar name="email recipient1" icon={UserIcon} />
+                    <Avatar name="email recipient2" icon={UserIcon} />
+                    <Avatar name="email recipient3" icon={UserIcon} />
+                  </AvatarGroup>
+                  <Text fontWeight="fontWeightBold">+6,078</Text>
+                </Box>
+              </Box>
+            </VisualPickerCheckbox>
+        </VisualPickerCheckboxGroup>
+    );
+}
+render(
+    <VisualPickerExample/>
+)
+`.trim();
+
+export const VisualPickerCheckboxHorizontal = `
+const VisualPickerExample = () => {
+    const [checked1, setChecked1] = React.useState(false)
+    const [checked2, setChecked2] = React.useState(false)
+    const [checked3, setChecked3] = React.useState(false)
+    const handleChange1 = (event) => setChecked1(event.target.checked)
+    const handleChange2 = (event) => setChecked2(event.target.checked)
+    const handleChange3 = (event) => setChecked3(event.target.checked)
+    return (
+        <VisualPickerCheckboxGroup
+            orientation="horizontal"
+            name={useUID()}
+            id={useUID()} 
+            legend="Select a channel to test"
+            helpText="The test message will only be sent to test recipients."
+        >
+            <VisualPickerCheckbox id={useUID()} labelText="sms" checked={checked1} onChange={handleChange1}>
+              <Box display="flex" flexDirection="column" rowGap="space30">
+                <Text as="span" fontWeight="fontWeightSemibold" fontSize="fontSize60">
+                  SMS
+                </Text>
+                <Text as="span" fontWeight="fontWeightNormal" fontSize="fontSize30">
+                  30 test recipients
+                </Text>
```

Comment:
chore: set the second line of text to colorTextWeak, along with the other items.